src/core/configManager.js

```diff
@@ -311,8 +313,9 @@ module.exports = class ConfigManager {
 
     async __publishTalentDiscoveryError(discovery, code) {
         try {
+            const rulesHash = jsonHash(discovery.config.rules)
```

Comment:
Does this mean that the config rules are hashed? -> Will the hash for the same rule (e.g. Speed 'IS SET') be static?